vault/resource_approle_auth_backend_role_secret_id.go

```diff
@@ -147,6 +154,8 @@ func approleAuthBackendRoleSecretIDCreate(d *schema.ResourceData, meta interface
 	} else {
 		data["metadata"] = ""
 	}
+	var reuseWrappingToken bool
+	reuseWrappingToken = d.Get("reuse_wrapping_token").(bool)
```

Comment:
I think we can safely omit the boolean declaration, replacing it with:
```suggestion
	reuseWrappingToken := d.Get("reuse_wrapping_token").(bool)
```